server/queue/queue-rabbitmq/src/main/java/org/apache/james/queue/rabbitmq/RabbitMQMailQueueManagement.java

```diff
@@ -54,6 +55,21 @@
 
     public void deleteAllQueues() {
         api.listQueues()
-            .forEach(queue -> api.deleteQueue("/", queue.getName()));
+            .forEach(queue -> api.deleteQueue(VHOST, queue.getName()));
+    }
+
+    public Long getQueueSize(MailQueueName name) {
```

Comment:
I think it's better to return an Optional<Long>, then you can fallback to use `mailQueueView.getSize()` in case of this method returns an empty Optional